invenio_jsonschemas/config.py

```diff
@@ -33,8 +33,8 @@
 JSONSCHEMAS_URL_SCHEME = 'https'
 """Default url scheme for schemas."""
 
-JSONSCHEMAS_REPLACE_REFS = False
-"""Whether to resolve $ref before serving a schema."""
-
 JSONSCHEMAS_LOADER_CLS = None
 """Loader class used in ``JSONRef`` when replacing ``$ref``."""
+
+JSONSCHEMAS_TRANSFORM = []
```

Comment:
We could put here import strings so that anybody can write their own transform function, instead of limiting to the ones provided in this package.